app/views/posts/_comment_thumb.html.erb

```diff
@@ -1,4 +1,8 @@
 <div class="comment-content">
     <span><%= commenter %>:</span>
     <span><%= comment %></span>
+    <%= link_to "Delete", user_post_comment_path(@post.author, @post, comment_id), data: {
+            turbo_method: :delete,
+            turbo_confirm: 'Are you sure you want to delete this comment?'
+        }, class: 'like-button' if can? :destroy, Comment %>
```

Comment:
- [ ] The delete button for comments is displayed regardless of the user's role. Kindly note that only the creator of the comment and the admin should be able to see the delete buttons at all. Kindly fix your views.